sdk/containerinstance/Microsoft.Azure.Management.ContainerInstance/src/Microsoft.Azure.Management.ContainerInstance.csproj

```
@@ -1,25 +1,25 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-	<!-- Please do not move/edit code below this line -->
-	<Import Project="$(RepoEngPath)/mgmt/AzSdk.reference.props" />
-	<!-- Please do not move/edit code below this line -->
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$(RepoEngPath)/mgmt/AzSdk.reference.props" />
+  <!-- Please do not move/edit code below this line -->
 
-	<PropertyGroup>
-		<Description>Microsoft Azure Container Instance Management Library</Description>
-		<Version>2.0.0</Version>
-		<AssemblyName>Microsoft.Azure.Management.ContainerInstance</AssemblyName>
-		<PackageId>Microsoft.Azure.Management.ContainerInstance</PackageId>
-		<PackageTags>Microsoft Azure Container Instance Management;Container Instance;</PackageTags>
-		<PackageReleaseNotes>
-			<![CDATA[
-        This is a release of the Azure Container Instance SDK. Included with this release is MSI identity operations.
+  <PropertyGroup>
+    <Description>Microsoft Azure Container Instance Management Library</Description>
+    <Version>3.0.0</Version>
+    <AssemblyName>Microsoft.Azure.Management.ContainerInstance</AssemblyName>
+    <PackageId>Microsoft.Azure.Management.ContainerInstance</PackageId>
+    <PackageTags>Microsoft Azure Container Instance Management;Container Instance;2019-12-01;</PackageTags>
```

```suggestion
    <PackageTags>Microsoft Azure Container Instance Management;Container Instance;</PackageTags>
```
No need to add API tags here because they are already imported from `AzSdk.RP.props`